scripts/setup_sgtm_tasks_project.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+
+import argparse
+from dataclasses import dataclass
+from typing import List
+
+import asana
+import sys
+
+
+@dataclass
+class EnumOption:
+    name: str
+    color: str
+
+
+@dataclass
+class CustomField:
+    name: str
+    enum_options: List[EnumOption]
+
+
+CUSTOM_FIELDS = [
+    CustomField(
+        name="PR Status",
+        enum_options=[
+            EnumOption(name="Open", color="green"),
+            EnumOption(name="Merged", color="purple"),
+            EnumOption(name="Closed", color="red"),
+        ],
+    ),
+    CustomField(
+        name="Build",
+        enum_options=[
+            EnumOption(name="Success", color="green"),
+            EnumOption(name="Failure", color="red"),
+        ],
+    ),
+]
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="""A helper script to set up an SGTM tasks Asana project with the suggested custom fields
+		"""
+    )
+
+    parser.add_argument(
+        "-p",
+        "--personal_access_token",
+        help="""The Asana Personal Access Token that will be used to set up the Asana project. If you do not already have one,
+		follow the instructions at https://developers.asana.com/docs/personal-access-token
+		""",
+        type=str,
+        required=True,
+    )
+
+    parser.add_argument(
+        "-n",
+        "--new_project_name",
+        help="""The Asana Project name - use this if you want to set up a project from scratch
+		""",
+        type=str,
+        required=False,
+    )
+
+    parser.add_argument(
+        "-t",
+        "--team_id",
```

Comment:
What's a good way for users to figure out this ID?

Reply:
If you click on the team page in Asana, you can get it from the URL. Should I add that to the help?